.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - 3.6
+services:
+  - postgresql
+addons:
+  chrome: stable
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.yarn-cache
+    - node_modules
+before_install:
+  - nvm install 10
+  - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+install:
+  - pip install cookiecutter
+  - pip install pip-tools
+  - yarn global add @angular/cli
+script:
+  - cookiecutter . --no-input
+  - cd dighum_pro
+  - cat bootstrap.log
+  - cd backend && pip-compile && cd ..
+  - cd functional-tests && pip-compile && cd ..
+  - yarn
+  - yarn django migrate
+  - yarn test-back && yarn test-front
```

Comment:
To make `yarn test-func` work, you need to run `yarn static-p` in advance, then run `yarn serve-p` in the background, then wait about a second for both servers to be ready and finally kick off `yarn test-func`. You might need to manually kill the background task in order for Travis to consider the build complete.

Reply:
I already tried this (well apart from `yarn static-p`), it doesn't work because the web drivers fail:
> selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home

https://travis-ci.com/UUDigitalHumanitieslab/cookiecutter-webapp-deluxe/jobs/192507563

Reply:
You might just need to `yarn global add chromedriver geckodriver` before running `yarn test`.

By the way, if you do `yarn test`, the unittests run first so you don't need to fiddle with a 1-second timeout. You still need to get hold of the pid of `yarn serve-p` so you can stop it, though.